aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -96,6 +96,10 @@ Tensor isfinite(const Tensor& self) {
   });
 }
 
+Tensor isinf(const Tensor& self) {
+  return at::native::isfinite(self).logical_not();
```

Comment:
Can we please use a kernel, similar to `isfinite`?

Reply:
what’s the reason for that versus implementing this in terms of `isfinite`? is it just to not have to do the `logical_not`?

Reply:
1 vs 2 kernels :)